buildrunner/config.py

```diff
@@ -0,0 +1,405 @@
+"""
+Copyright 2021 Adobe
```

Comment:
Copyright 2022?

Reply:
the linter does not like that - it is looking for `2021`